config/horizon.json

```diff
@@ -45,6 +46,7 @@
 		"happychat": false,
 		"help": true,
 		"help/courses": true,
+		"home/layout-dev": true,
```

Comment:
Please create a server patch instead to remove the check for ?dev

Reply:
The update should be removing this block in wp-content/lib/home/views.php. Let us know if you have trouble. Since the dev query param will be used for general Home development we won't want to turn this on for any environment except Calypso development.
```
case VIEW_EDITOR_DEPRECATION:
--
// Not ready for production yet.
if ( ! $dev ) {
return false;
}
```